csharp/ql/test/query-tests/Security Features/CWE-022/AssemblyPathInjection/Test.cs

```diff
@@ -0,0 +1,23 @@
+// semmle-extractor-options: /nostdlib /noconfig /r:${env.windir}\Microsoft.NET\Framework64\v4.0.30319\mscorlib.dll /r:${env.windir}\Microsoft.NET\Framework64\v4.0.30319\System.Web.dll /r:${env.windir}\Microsoft.NET\Framework64\v4.0.30319\System.dll
```

Comment:
These tests won't run on Linux. We need to find an equivalent set of assemblies that work on .NET Core. I think it'll be best if I create a pull request for this.

Reply:
Let me know if you want me to remove the test for now and try to create a simple stub instead